com.woltlab.wcf/templates/__totpNewDeviceContainer.tpl

```diff
@@ -0,0 +1,44 @@
+<section id="{@$container->getPrefixedId()}Container"{*
+	*}{if !$container->getClasses()|empty} class="{implode from=$container->getClasses() item='class' glue=' '}{$class}{/implode}" {/if}{*
```

Comment:
```suggestion
	*}{if !$container->getClasses()|empty} class="{implode from=$container->getClasses() item='class' glue=' '}{$class}{/implode}"{/if}{*
```

Reply:
This is a common issue in all containers. I'll fix that in master.